nl/guides/exercise-series-management/README.md

```diff
@@ -0,0 +1,272 @@
+
+Klik op de oefeningenreeks in de [navigatiebalk](../for-students#navigatiebalk) om het aanpassen te annuleren.
+
+![reeks bewerken stoppen](./staff.series_edit_cancel.png)
+
+Onderaan de pagina vind je de activiteiten die reeds tot deze reeks behoren en mogelijke activiteiten om eraan toe te voegen. Klik op de toevoegknop aan de rechterkant van een oefening om de oefening aan de oefeningenreeks toe te voegen.
+
+![reeks oefening toevoegen](./staff.series_add_exercise.png)
+
+Onder de hoofding `Oefeningen in deze reeks` kan je aan de rechterkant van een oefening op de verwijderknop klikken om de oefening uit de oefeningenreeks te verwijderen.
+
+![reeks oefening verwijderen](./staff.series_remove_exercise.png)
+
+Versleep de verplaatsknop aan de linkerkant van de oefeningen om de volgorde van de oefeningen aan te passen. De volgorde waarin de oefeningen onder de hoofding `Oefeningen in deze reeks` gerangschikt worden, is immers ook de volgorde waarin de oefeningen weergegeven worden in de oefeningenreeks.
+
+![oefening verplaatsen](./staff.series_move_exercise.png)
+
+Via de zoekbalk kan je bestaande oefeningen filteren op naam, beschikbare vertalingen, programmeertaal, labels, repository of type.
+
+![oefeningen zoeken](./staff.series_search_exercises.png)
+
+::: tip Belangrijk
+
+We veronderstellen hier dat de oefeningen die aan de oefeningenreeks moeten gekoppeld worden reeds beschikbaar zijn in Dodona. Het opstellen, publiceren en delen van oefeningen wordt [hier](/nl/new-exercise-repo) besproken.
+
+:::
+
+Klik op de afwerkknop in de rechterbovenhoek van het paneel om de bewerkingen te bewaren. Dit is enkel van toepassing op de reeks-eigenschappen. Bewerkingen op de oefeningen binnenin die reeks worden meteen automatisch opgeslagen.
+
+![reeks bewerken opslaan](./staff.series_edit_submit.png)
+
+Na het afwerken navigeer je terug naar de oefeningenreeks in het leerpad, waar je kunt vaststellen dat de nieuwe eigenschappen van de reeks meteen van toepassing zijn.
+
+
+## Oefeningenreeks verwijderen
+
+Uiteraard is het mogelijk om een reeks te verwijderen uit een cursus. De actie vind je analoog aan het bewerken in het reeksen-beheren menu of in het reeks-actiesmenu.
+
+![reeks verwijderen](./staff.series_delete.png)
+
+## Oefeningenreeks herordenen
+
+Het kan handig zijn om reeksen in een cursus een bepaalde volgorde te geven, om ze bijvoorbeeld te sorteren volgens moeilijkheidsgraad. Standaard zullen ze gesorteerd worden in omgekeerd chronologische volgorde op basis van wanneer je ze toevoegt. Zo moeten studenten minder scrollen als ze een reeks willen maken. In het [reeksen-beherenpaneel](#oefeningenreeks-aanmaken) kan je in de tabel van de reeds toegevoegde reeksen ze verslepen via het icoontje aan de linkerkant.
+
+## Oefeningenreeks beheren
+
+Dit menu bevat een aantal handige acties die cursusbeheerders kunnen uitvoeren op een reeks. Naast bewerken en verwijderen zijn er nog enkele mogelijkheden:
```

Comment:
explain that the menu can be accessed by clicking on the 3 dots on the top right corner of a series